src/algorithms/excitation/chepigaansatz.jl

```diff
@@ -1,11 +1,11 @@
-
 """
-    ChepigaAnsatz <: Algorithm
+$(TYPEDEF)
 
-Optimization algorithm for excitations on top of MPS groundstates, as
-introduced in this [paper](https://doi.org/10.1103/PhysRevB.96.054425).
+Single-site optimization algorithm for excitations on top of MPS groundstates.
 
 ## Fields
+
+$(TYPEDFIELDS)
 - `alg::A = Defaults.eigsolver`: algorithm to use for the eigenvalue problem.
```

Comment:
```suggestion
```

Reply:
Also, it might be nice to type restrict `A<:KrylovKit.KrylovAlgorithm` to have an accurate type annotation for the `alg` field?

Reply:
I think this could be okay for now, but in principle I don't want to actually restrict the type, since there really is no need for that. On the other hand, probably no-one is really complaining that if they really want a different eigensolver type it has to subtype the krylovkit algorithm type so I'll change it until further complaints :)